src/CalcManager/UnitConverter.h

```diff
@@ -145,10 +141,6 @@ namespace UnitConversionManager
         {
         }
 
-        virtual ~ConversionData()
```

Comment:
It doesn't appear that anything is supposed to inherit from `ConversionData` so there is no need to have a virtual destructor which adds overhead to this type by giving it a vtable.